rotkehlchen/chain/ethereum/modules/curve/decoder.py

```diff
@@ -34,7 +46,47 @@ def __init__(
             msg_aggregator=msg_aggregator,
             native_currency=A_ETH,
             aave_pools=AAVE_POOLS,
-            curve_deposit_contracts=CURVE_DEPOSIT_CONTRACTS,
-            curve_swap_router=CURVE_SWAP_ROUTER,
-            gauge_controller=GAUGE_CONTROLLER,
+            curve_deposit_contracts=CURVE_DEPOSIT_CONTRACTS | {DEPOSIT_AND_STAKE_ZAP},
+            curve_swap_routers={CURVE_SWAP_ROUTER, CURVE_SWAP_ROUTER_NG},
         )
+
+    def _decode_curve_gauge_votes(self, context: DecoderContext) -> DecodingOutput:
+        if context.tx_log.topics[0] != GAUGE_VOTE:
+            return DEFAULT_DECODING_OUTPUT
+
+        user_address = hex_or_bytes_to_address(context.tx_log.data[32:64])
+        if not self.base.is_tracked(user_address):
```

Comment:
I know you just moved it but let's improve

```suggestion        
        if not self.base.is_tracked(user_address := hex_or_bytes_to_address(context.tx_log.data[32:64])):
```